source/manual/concept-of-stacks.html.md

```diff
@@ -1,44 +0,0 @@
----
```

Comment:
Stacks were a nice idea but ultimately didn't go any further than `blue` and they now just get in the way.

Reply:
Interestingly, after putting this up I notice we've got 'green' DNS records defined on integration. But I still don't think it affects anything day-to-day.